containers/reverse-proxy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM envoyproxy/envoy-alpine:v1.14.4
+
+ARG SERVICE_PORT
+ARG PORT0 
```

Comment:
We should just assume these are in the container environment rather than passing them in as args at image-build time.

Reply:
Done